samples/can/Kconfig

```diff
@@ -0,0 +1,93 @@
+# Kconfig - Private config options for can sample app
+
+#
+# Copyright (c) 2018 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "Controller Area Network sample application"
+
+config ZEPHYR_BASE
+	string
+	option env="ZEPHYR_BASE"
+
+source "$ZEPHYR_BASE/Kconfig.zephyr"
+
+config CAN_DEV
+	string "Name of the CAN device"
+	default "CAN_1"
+	help
+		"Name of the can device used for send an receive."
```

Comment:
The help texts do not follow zephyr style. Also noticed that some of the help text have mismatch ". Anyway, there is no need to put help text inside "". So you can have it like this:

```
	help
	  Name of the can device used for send an receive.

```